docs/wiki/[User-Guide]-Getting-Started.md

```diff
@@ -1,4 +1,4 @@
-## Considerations
+## Initial Considerations
```

Comment:
Should we also ad a link to the dedicated "considerations" page? Or could the considerations you've documented just be added to this page instead of creating a new page?

Reply:
I initially used a separate page as these guidelines were pertaining to the deployment of resources within landing zones, as opposed to the platform resources that come with ESLZ.

Given there aren't many of them I'll see if I can add them to the getting started page under a distinct heading, simplifying the change.

Reply:
Simplified and added to getting started page